mltsp/tests/test_custom_feats.py

```diff
@@ -87,6 +88,8 @@ def test_execute_functions_in_order():
     assert(all(x in feats_list[0] for x in ["a", "l", "o"]))
 
 
```

Comment:
Suggestion:

```no_docker = (os.getenv("MLTSP_NO_DOCKER") == "1")```

and then later on

```@skipIf(no_docker)```

Also, let's use the slightly more sophisticated implementation of ``skipif``:

```
from numpy.testing import decorators as dec

@dec.skipif(no_docker, "Docker testing turned off")
...
```